korgan/claim_ten_test_gate.py

```
@@ -0,0 +1,477 @@
+        t4_parts.append("не денежный иск о взыскании долга — четырехпозиционный тест не применяется")
+
+    inventions = _unknown_party_facts(case_context, draft)
+    duplicate_transition = any(_TRANSITION_RE.search(str(item)) for item in [*draft.facts, *draft.legal_basis, *draft.requests])
+
+    ledger = build_claim_money_ledger(list(draft.requests or []))
+    price = parse_amount_kzt(draft.price_of_claim or "")
+    marked_gap = _DATA in prayer or _VERIFY in prayer or _DATA in (draft.price_of_claim or "") or _VERIFY in (draft.price_of_claim or "")
+    arithmetic_ok = bool(marked_gap or (not ledger.unresolved_requests and ((ledger.total == 0 and price is None) or ledger.total == price)))
+    if debt and _request_has(_PENALTY_RE, draft) and not marked_gap:
+        penalty_calc = "\n".join([*draft.facts, draft.late_interest or ""])
+        arithmetic_ok &= bool(re.search(r"(?i)(?:расч[её]т|×|дн\.|дней|календарн\w*\s+дн)", penalty_calc) and _PENALTY_RE.search(penalty_calc))
```

Ruff (0.16.2)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)